### DISCUSSION TITLE
Is there a way to ignore peer dependencies of a particular package (not packages that themselves are peer deps)

### DISCUSSION THREAD

**Discussion by yuchant** [pnpm/pnpm#5139] [Q&A] [answered]
Right now I have Storybook as a dep, but this package only defines stories and does not launch Storybook itself, so I don't need the peer deps.

```
"peerDependencyRules": {
      "ignoreMissing": [
        "@babel/*",
      ]
}
```

Is there a way to ignore the peer deps for a package? Say all peer deps of `@storybook/react`

**Comment by zkochan** [comment, marked as answer]
no, it is only possible to list the peer dependencies that you want to ignore.

**Reply by yuchant** [reply]
Thank you!
